### PR TITLE
adding a page for seL4-related jobs

### PR DESCRIPTION
**Pull request by june-andronick**
Signed-off-by: June Andronick <june@sel4.systems>

### CODE REVIEW

**Comment by june-andronick** [comment]
Once the page is online, we can add a news item saying that it's online. Then we can add another news item with the message to the community about ways to support seL4.

File: content/Foundation/Jobs/home.pml [hunk 1]
```diff
@@ -0,0 +1,84 @@
+<!--<title>Jobs about seL4>-->
+<!--
+    Copyright 2020 seL4 Project a Series of LF Projects, LLC.
+    SPDX-License-Identifier: CC-BY-SA-4.0
+-->
+<h1>
+  Jobs about
```
Comment (lsf37) [inline content/Foundation/Jobs/home.pml:7]:
```suggestion
  Jobs around
```
?

File: content/Foundation/Jobs/home.pml [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+This page matches organisations looking for seL4 expertise  and people looking for
+jobs in the seL4 ecosystem.
+</p>
+
+<p>
+The list is in chronological order, more recent at the top.
+</p>
+
+<p>
+If you are an organisation wanting to post here, email <a
```
Comment (lsf37) [inline content/Foundation/Jobs/home.pml:21]:
```suggestion
If you are an organisation and would like to post here, email <a
```

File: content/Foundation/Jobs/home.pml [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    href="mailto:foundation@sel4.systems"
+    class="email">foundation@sel4.systems</a> with:
+</p>
+
+<ul>
+<li>
+<b> Company description and position in seL4 ecosystem</b>: short (few
```
Comment (lsf37) [inline content/Foundation/Jobs/home.pml:28]:
```suggestion
<b> Company description and position in seL4 ecosystem</b>: short (a few
```

File: content/Foundation/Jobs/home.pml [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+sentences) description of the company's profile and its role or
+interest in the seL4 ecosystem.
+</li>
+<li>
+<b> Position details and link to seL4</b>: short (few sentences)
```
Comment (lsf37) [inline content/Foundation/Jobs/home.pml:33]:
```suggestion
<b> Position details and link to seL4</b>: short (a few sentences)
```

**Review by lsf37 — APPROVED** [review]
Looks good. It would be nice to start with one from the foundation itself :-)

**Comment by gernotheiser** [comment]
Obviously this needs to be linked somewhere from at least the Foundation page.
But I thought we promised this as a service to members? The present text has it open to everyone.
Suggest s/Jobs about seL4/Jobs in the seL4 ecosystem

**Comment by june-andronick** [comment]
Gernot:
- would you have info to add a first job ad for the support position (noting that we're trying to get budget to make it FT)?
- I forgot that we wanted to limit this to members. Good point. Should I just say on the page that it's limited to members? (keeping it here as an incentive to join)

**Comment by june-andronick** [comment]
@lsf37 I accepted to changes from the browser and this seems to be causing problems with signatures and commit title being too short. Not sure how to fix.

**Comment by lsf37** [comment]
> @lsf37 I accepted to changes from the browser and this seems to be causing problems with signatures and commit title being too short. Not sure how to fix.

Yes, that's the annoying bit with the browser support, it doesn't know about the sign-off. You can either rebase and fix up the commit message, or you can use "Squash and merge" in the browser and add a Signed-off-by line manually in the commit message there. I've set the DCO to "pass", so the merge option should now be enabled.

**Pull request merged**